.github/workflows/webapp_unit_tests.yml

```diff
@@ -0,0 +1,24 @@
+name: "Webapp Unit Tests"
```

Comment:
Added this so that unit tests will run automatically on every PR (like the BE ones).